modules/getting-started/proc-techdocs-configure-odf-helm.adoc

```diff
@@ -0,0 +1,19 @@
+[id="proc-techdocs-configure-odf-helm_{context}"]
+
+= Adding access information to the {product} Helm chart
```

Comment:
I know this is the headline I proposed, but I still think it is a bit weird, and I don't know how to improve it :-(. In this section, we are configuring Helm Chart in a way that makes all information (that is stored in Secret and ConfigMap) about ObjectBucket available in the container as environment variables. Those variables are then used in TechDocs plugin configuration